### DISCUSSION TITLE
Fifo pipe with Docker

### DISCUSSION THREAD

**Discussion by kerta1n** [mikebrady/shairport-sync#1619] [Q&A] [answered]
Hello, I have Shairport as a container running along with the djmaze/librespot-snapserver container which (from the name) combines the two into one container. I figured passing a pipe between the two containers would be easiest, but from what I read from the [example conf](https://github.com/mikebrady/shairport-sync/blob/54cb9de1836fcb764bbae7397b6f79899f674268/scripts/shairport-sync.conf#L157), shairport has to be built with the pipe flag. Do the provided images have this, or do I need to build the image myself? And if so, how would I pass that into the Dockerfile? Maybe I'm missing something and the /tmp/shairport-sync-audio can be enabled from Docker compose (I used composerize.com), but I didn't see anything in the /tmp dir in the container. Great project, kudos to the devs, thanks.

**Comment by charlesomer** [comment, marked as answer]
The docker images are built with pipe: [here](https://github.com/mikebrady/shairport-sync/blob/6240fd83707b77fe2fb2250f8136e530e36010d2/docker/Dockerfile#L59) 

Once enabled in the config, you should then then be able to expose the pipe to the host via docker compose volumes
